src_tests/unittests/osm_configurator/view/states/test_state.py

```diff
@@ -0,0 +1,92 @@
+import pytest
+
+import src.osm_configurator.view.states.state as state_i
+import src.osm_configurator.view.states.positioned_frame as positioned_frame_i
+import src.osm_configurator.view.toplevelframes.top_level_frame as top_level_frame_i
+import src.osm_configurator.view.states.state_manager as state_manager_i
+import src.osm_configurator.view.states.state_name_enum as state_name_enum_i
+
+
+@pytest.fixture
+def positioned_frame():
```

Comment:
Please add comments for better readability

Reply:
Tests have recieved comments